src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/Html/DefaultDisplayTemplates.cs

```diff
@@ -266,11 +256,16 @@ public static IHtmlContent ObjectTemplate(IHtmlHelper htmlHelper)
                     readOnly: true,
                     additionalViewData: null);
 
-                content.Append(templateBuilder.Build());
```

Comment:
stick `templateBuilder.Build()` into a variable to make it blindingly obvious it's called without regard to `HideSurroundingHtml`.